pkg/api/types.go

```diff
@@ -767,9 +771,9 @@ const (
 	// The default compute resource namespace for all standard resource types.
 	DefaultResourceNamespace = "kubernetes.io"
 	// CPU, in cores. (500m = .5 cores)
-	ResourceCPU ResourceName = "cpu"
+	ResourceCPU ResourceName = DefaultResourceNamespace + "/cpu"
```

Comment:
cc @derekwaynecarr for the impact on admission control. It looks like it was already using these strings:
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/plugin/pkg/admission/limitranger/admission.go#L150

Hmm. Maybe we should just reserve unnamespaced resources for Kubernetes. Otherwise, defaulting, conversion, validation, comparisons, etc. are pretty complicated and error-prone.

@thockin WDYT?